components/brave_wallet_ui/page/screens/fund-wallet/fund-wallet.tsx

```diff
@@ -314,24 +309,24 @@ export const FundWalletScreen = () => {
 
                 {assetsForFilteredNetwork.length
                   ? <TokenLists
-                      enableScroll
-                      maxListHeight='38vh'
-                      userAssetList={assetsForFilteredNetwork}
-                      networks={networksFilterOptions}
-                      hideAddButton
-                      hideAssetFilter
-                      hideAccountFilter
-                      hideAutoDiscovery
-                      estimatedItemSize={100}
-                      renderToken={renderToken}
-                    />
+                    enableScroll
+                    maxListHeight='38vh'
+                    userAssetList={assetsForFilteredNetwork}
+                    networks={networksFilterOptions}
+                    hideAddButton
+                    hideAssetFilter
+                    hideAccountFilter
+                    hideAutoDiscovery
+                    estimatedItemSize={100}
+                    renderToken={renderToken}
+                  />
                   : <Column>
-                      <LoadingIcon
-                        opacity={1}
-                        size='100px'
-                        color='interactive05'
-                      />
-                    </Column>
+                    <LoadingIcon
+                      opacity={1}
+                      size='100px'
+                      color='interactive05'
+                    />
+                  </Column>
```

Comment:
indentation seems off here. not seeing adjustments like this when I format the file myself

Reply:
Hmmm yes, does seem weird.. I'll look into my setting. Undid this formatting.